### ISSUE TITLE
Traceback occurred

### ISSUE THREAD

**Issue by dalevova** [requests/requests-ntlm#69]
For version 0.3.0 traceback occurred (for 0.2.0 works as expected)

Headers:
Status code: 401 Unauthorized
WWW-Authenticate:"Basic realm="data""

`File "/env/lib/python2.7/site-packages/requests/api.py", line 71, in get
    return request('get', url, params=params, **kwargs)
  File "/env/lib/python2.7/site-packages/requests/api.py", line 57, in request
    return session.request(method=method, url=url, **kwargs)
  File "/env/lib/python2.7/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/env/lib/python2.7/site-packages/requests/sessions.py", line 591, in send
    r = dispatch_hook('response', hooks, r, **kwargs)
  File "/env/lib/python2.7/site-packages/requests/hooks.py", line 31, in dispatch_hook
    _hook_data = hook(hook_data, **kwargs)
  File "/env/lib/python2.7/site-packages/requests_ntlm/requests_ntlm.py", line 130, in response_hook
    'Authorization', r, auth_type, kwargs)
UnboundLocalError: local variable 'auth_type' referenced before assignment`

**Comment by Lukasa** [comment]
Yup, so that traceback is bad. However, it reveals a problem in your code: you're not getting an NTLM challenge, you're getting a Basic auth challenge. Why are you using requests-ntlm in that case?

**Comment by rbcarson** [comment]
It looks like #73 resolved this, right?
